src/view/command_reference_sidebar.js

```diff
@@ -0,0 +1,61 @@
+/*
+ */
+
+var React = require("react");
+var Constants = require("../constants");
+var ResourceLoader = require("../util/resource_loader");
+var Sidebar = require('react-sidebar').default;
+
+var CommandReferenceSidebar = React.createClass({
+  styles: {
+    root: {
+      fontFamily: '"HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif',
```

Comment:
I think for now you can leave the fontFamily unspecified here, and we can standardize fonts across the whole interface later using CSS classes. We'll probably have to ship our fonts for consistency, so that we do not rely on the locally installed font set.